<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>771</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>771</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Almaz SQL Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="regist_tab">
       <attribute name="title">
        <string>Регистрация</string>
       </attribute>
       <widget class="QLineEdit" name="host_lbl">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Хост:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>База данных:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="user_lbl">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>postgres</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="password_lbl">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vfcnth</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="database_lbl">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>190</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>first</string>
        </property>
       </widget>
       <widget class="QPushButton" name="connect_but">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>401</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Соединить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="disconnect_but">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>401</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Разорвать соединение</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Драйвер БД:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="DB_box">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>31</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>QOCI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QODBC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QMYSQL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QTDS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QPSQL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QSQLITE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QIBASE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDB2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Другой</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="DB_lbl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="query_tab">
       <attribute name="title">
        <string>Запрос</string>
       </attribute>
       <widget class="MyTextEdit" name="query_text">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>731</width>
          <height>161</height>
         </rect>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="query_but">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>731</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Запросить (Shift+Enter)</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="result_table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>731</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>mytextedit.h</header>
   <slots>
    <signal>mySignal()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
